### PR TITLE
Obtain default thresholds for stream

### PR DESCRIPTION
**Pull request by evulins**
Now Threshold configurator displays real default values for thresholds

[Trello card ](https://trello.com/b/LDFwhuRa/aircasting-website)

### CODE REVIEW

File: app/javascript/react/store/thresholdSlice.ts [hunk 1]
```diff
@@ -8,10 +8,10 @@ interface ThresholdState extends Thresholds {}
 
 export const initialState: ThresholdState = {
   min: 0,
-  low: 30,
-  middle: 50,
-  high: 80,
-  max: 120,
+  low: 0,
+  middle: 0,
+  high:0,
```
Comment (jankrzempek) [inline app/javascript/react/store/thresholdSlice.ts:13]: space 😄

File: app/serializers/fixed_stream_serializer.rb [hunk 2]
```diff
@@ -19,6 +19,11 @@ def serialized_stream(stream)
       unit_symbol: stream.unit_symbol,
       update_frequency: '1 minute',
       last_update: stream.session.last_measurement_at,
+      min: 0,
+      low: 20,
+      middle: 40,
+      high: 60,
+      max: 100,
```
Comment (jankrzempek) [inline app/serializers/fixed_stream_serializer.rb:26]: should this be included here at the end? The idea was to mock it until the api is not ready

**Review by jankrzempek — APPROVED** [review]
So FAAAST 🚀

File: app/serializers/fixed_stream_serializer.rb (hunk 2 above)
Comment (evulins) [reply app/serializers/fixed_stream_serializer.rb:26]: Yes, I will delete it once the backend is ready. I left it for testing purposes.

**Pull request merged**